packages/fxa-payments-server/src/routes/Subscriptions/ActionButton.tsx

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { Customer } from '../../store/types';
+
+import * as PaymentProvider from '../../lib/PaymentProvider';
+
+export type ActionButtonProps = {
+  customer: Customer;
+  onRevealUpdateClick: () => void;
+  revealFixPaymentModal: () => void;
+};
+
+export const ActionButton = ({
+  customer,
+  onRevealUpdateClick,
+  revealFixPaymentModal,
+}: ActionButtonProps) => {
+  const {
+    billing_agreement_id,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const stripeActionButton = () => (
+    <button
+      data-testid="reveal-payment-update-button"
+      className="settings-button"
+      onClick={onRevealUpdateClick}
+    >
+      <Localized id="pay-update-change-btn">
+        <span className="change-button" data-testid="change-button">
+          Change
+        </span>
+      </Localized>
+    </button>
+  );
+
+  const paypalActionButton = () => (
+    <Localized id="pay-update-change-btn">
+      <LinkExternal
+        data-testid="change-payment-update-button"
+        className="settings-button"
+        href={`https://www.paypal.com/myaccount/autopay/connect/${billing_agreement_id}`}
```

Comment:
Re: [this comment](https://github.com/mozilla/fxa/pull/7909#discussion_r599185034) for QA: I was wrong on the URL not working for sandbox PayPal. If you modify it before clicking the link, i.e. to "https://www.sandbox.paypal.com/myaccount/autopay/connect/${billingAgreementId}", it does work. Not sure what happened before...

Reply:
@biancadanforth  If that's the case, then we shouldn't hard coed the `href` here correct?  We'd like QA to use the sandbox when they are testing on stage.

Reply:
Yes if there's a good place to set the domain here based on whether we're in dev/stage (https://www.sandbox.paypal.com) or prod (https://www.paypal.com), that'd be a good change.

Reply:
Looks like we could use `apiUrl` https://github.com/mozilla/fxa/blob/351fa2b439c323ef5199efcf78790b3aee8a4a2e/packages/fxa-payments-server/server/config/index.js#L355